Python/flowgraph.c

```diff
@@ -261,8 +261,8 @@ dump_instr(cfg_instr *i)
     if (HAS_TARGET(i->i_opcode)) {
         sprintf(arg, "target: %p [%d] ", i->i_target, i->i_oparg);
     }
-    fprintf(stderr, "line: %d, opcode: %d %s%s\n",
-                    i->i_loc.lineno, i->i_opcode, arg, jump);
+    fprintf(stderr, "line: %d, opcode: %d (%s) %s%s\n",
+                    i->i_loc.lineno, i->i_opcode, _PyOpcode_OpName[i->i_opcode], arg, jump);
```

Comment:
Yes, I did this on my local branch only yesterday.

How about:

```suggestion
    fprintf(stderr, "line: %d, %s [%d]  %s%s\n",
                    i->i_loc.lineno, _PyOpcode_OpName[i->i_opcode], i->i_opcode, arg, jump);
```